apps/storybook/package.json

```diff
@@ -1,17 +1,17 @@
 {
   "name": "@repo/storybook",
-  "version": "0.0.9",
+  "version": "0.1.0",
   "type": "module",
   "private": true,
   "scripts": {
     "dev": "storybook dev -p 6006 --no-open",
     "build": "storybook build",
-    "test": "playwright test",
+    "test": "playwright test --update-snapshots",
     "lint": "eslint stories/**/*.tsx components/**/*.tsx --max-warnings=0 && stylelint **/*.css --max-warnings=0 --allow-empty-input",
     "clean": "rm -rf .turbo && rm -rf node_modules",
     "serve-storybook": "http-server storybook-static -p 6006",
     "update-screenshots": "playwright test --update-snapshots",
-    "test-docker": "docker-compose -f ../../docker-compose.e2e.yml up"
+    "test-docker": "docker-compose -f ../../docker-compose.e2e.yml up --build"
```

Comment:
let's not add build flag here since it's relevant only for the first run, and then it will just slow things down, we can add a not about it in readme or add a separate command